content/en/tracing/guide/ddsketch_trace_metrics.md

```diff
@@ -0,0 +1,46 @@
+---
+title: DDSketch based Metrics in APM
+kind: guide
+aliases:
+- /tracing/faq/resource-trace-doesn-t-show-up-under-correct-service/
+further_reading:
+- link: "https://www.datadoghq.com/blog/engineering/computing-accurate-percentiles-with-ddsketch/"
+  tag: "Blog"
+  text: "Computing Accurate Percentiles with DDSketch"
+- link: "https://docs.datadoghq.com/metrics/distributions/"
+  tag: "Documentation"
+  text: "Learn more about Distributions"
+- link: "https://docs.datadoghq.com/tracing/guide/metrics_namespace/"
+  tag: "Documentation"
+  text: "Learn more about Trace Metrics"
+---
+
+Trace metrics are collected automatically for your services and resources and retained for 15 months. The latency percentiles exist as individual time series and in beta as a DD Distribution Metric. This means that we now offer more data in easier formats. Instead of having a different metric for each percentile, separately for services, resources and second primary tags, we now offer a simple metric:
+
+- `trace.<SPAN_NAME>`:
+  - *Prerequisite:* This metric exists for any APM service in beta.
+  - *Description:* Represents latency distributions for all services, resources and versions across different environments and second primary tags.
+  - *Metric type:* [DISTRIBUTION][1]
+  - *Tags:* `env`, `service`, `version`, `resource`, and [the second primary tag][2].
+
+The APM Service and Resource pages now use this new metric for customers in the Beta automatically. This means you can use these metrics to power your dashboards and monitors.
+
+**What does this metric mean for me?**
+- Distribution metrics offer new capabilities compared with the existing gauge metric like precise aggregation in time (instead of a rollup) and open the door for better visualizations and functionality in the future.
```

Comment:
- Distribution metrics offer new capabilities compared with the existing gauge metric like precise percentile calculation over time periods (instead of a rollup function) and open the door for better visualizations and functionality in the future.